### PR TITLE
Add location to f2f map

### LINKED ISSUE #760: Add location to F2F payment option
**Is your feature request related to a problem? Please describe.**
Currently, if i understand correctly, the F2F payment option has no possibility to set a location (eg City or coordinates). This renders the otherwise private and useful payment method fully useless as the trade partner cannot see where the other side is located before accepting the order and paying the bond. In Bisq for example you can see the location of the p2p trade before accepting the offer, so you can only accept it if the location is reachable/useful. This is probably also the reason why there are so few F2F offers active.

**Describe the solution you'd like**
Add a field to enter the location when creating an offer with F2F payment method and show the location in the orderbook either always (maybe if only F2F is accepted) or for example on hover over the F2F icon or when looking at the offer details.

**Additional context**
For example see Bisq where this is being done for a long time.
Bildschirmfoto vom 2023-07-29 23-27-18

### PR DESCRIPTION
**Pull request by KoalaSat**
## What does this PR do?
Fixes #760

This PR introduces the map location selector for Order creation and the map chart to visualize all book orders with coordinates.

## Checklist before merging
- [x] Install [pre-commit](https://pre-commit.com/#installation) and initialize it: `pip install pre-commit`, then `pre-commit install`. Pre-commit installs git hooks that automatically check the codebase. If pre-commit fails when you commit your changes, please fix the problems it points out.

### CODE REVIEW

File: nodeapp/docker-compose.yml [hunk 1]
```diff
@@ -25,7 +26,7 @@ services:
 
   tor:
     build: ../docker/tor
-    container_name: tor-dev
+    container_name: tor-dev-frontend
```
Comment (KoalaSat) [inline nodeapp/docker-compose.yml:29]: Changing names allows to have a docker group for `fullstack` and another docker group for `frontend`

File: docker-compose.yml [hunk 2]
```diff
@@ -119,7 +123,8 @@ services:
     network_mode: service:tor
 
   celery-beat:
-    image: backend
+    image: backend-image
+    pull_policy: never
     container_name: cele-beat-dev
     restart: always
     environment:
```
Comment (KoalaSat) [inline docker-compose.yml:130]: This seems to fix the problems in OSx, please @Reckless-Satoshi can you confirm I didn't break other OS?

File: mobile/html/Web.bundle/index.html [hunk 3]
```diff
@@ -11,6 +11,7 @@
     <link rel="stylesheet" href="css/fonts.css"/>
     <link rel="stylesheet" type="text/css" href="css/loader.css"/>
     <link rel="stylesheet" type="text/css" href="css/index.css"/>
+    <link rel="stylesheet" type="text/css" href="css/leaflet.css"/>
```
Comment (KoalaSat) [inline mobile/html/Web.bundle/index.html:14]: I didn't test it for mobile, I can try to build it tomorrow

**Comment by Reckless-Satoshi** [comment]
Hey @KoalaSat , this looks amazing! We can merge as is.

After merge I will:
1. add the new vector graphics to the self-hosted nodeapp (`/static/assets/vector` is in `.dockerignore`)
2. add text / pop ups to explain the privacy concerns with the tiling server and the randomization of the user lat/lon input
3. Fiddle around leafleft background and colors to try to make it match or dark/light themes when switched.
4. Once ready, we can cut version v0.5.3 and release.

Please post an invoice from a proxy node id and with an expiration time of +24h for **2.5 M Sats** tip from the development fund. Thank you!!

**Comment by KoalaSat** [comment]
````
lnbc25m1pjju4wgpp5lpjewvh6af7ar50dgnyw2snpxhaecfussgksmmw4jclmfj54r4eqdq5g9kxy7fqd9h8vmmfvdjscqzzsxqyz5vqsp5xz759lr2ekqqz89stacyn50cgx8y2qd4q7ta648tapttk704t72q9qyyssqwy82p3xqhdalz506knjr06fup62d5gy4attfuzesel9c68pm7c2yvxp0778hqz7cfd86vdufsdu2ras39jwc9x208w2hpgasdf044vcp0vkx6s
````

🚀

**Comment by Reckless-Satoshi** [comment]
> ```
> lnbc25m1pjju4wgpp5lpjewvh6af7ar50dgnyw2snpxhaecfussgksmmw4jclmfj54r4eqdq5g9kxy7fqd9h8vmmfvdjscqzzsxqyz5vqsp5xz759lr2ekqqz89stacyn50cgx8y2qd4q7ta648tapttk704t72q9qyyssqwy82p3xqhdalz506knjr06fup62d5gy4attfuzesel9c68pm7c2yvxp0778hqz7cfd86vdufsdu2ras39jwc9x208w2hpgasdf044vcp0vkx6s
> ```

d177a8a9246373c075588a1ab2e44c5580171acbce1195bdb039cb69318b7010

**Pull request merged**